Gemfile

```diff
@@ -96,10 +96,11 @@ gem 'active_scheduler', '~> 0.5.0'
 gem 'retriable', '~> 3.1'
 gem 'cld3', '~> 3.2.3'
 
-#Assets-related gems
+# Assets-related gems
 gem 'coffee-rails', '~> 4.2.2'
 gem 'uglifier', '~> 4.1.2'
-gem 'less-rails-bootstrap', '~> 3.3.5'
+# Using forked copy until https://github.com/metaskills/less-rails-bootstrap/pull/132 is merged
+gem 'less-rails-bootstrap', git: 'https://github.com/veelenga/less-rails-bootstrap', :ref => '7c479c2fdff500dc036c15364aa085332a73c642'
```

Comment:
@PatKelsh , for readability please:
- break this into 2 lines
- use ruby 1.9 hash syntax
```ruby
gem 'less-rails-bootstrap', git: 'https://github.com/veelenga/less-rails-bootstrap',
                            ref: '7c479c2fdff500dc036c15364aa085332a73c642'
```